.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# Required:
+------------
+[ ] I believe this isn't a duplicate topic
+[ ] This report is not related to an adapter
+
+For adapter related issues, please go to the appropriate repository - likely in
+the [OpenTimelineIO github organization](https://github.com/OpenTimelineIO/).
+For general questions and help please use the
+[Academy Software Foundation slack](https://slack.aswf.io/),
+[#opentimelineio](https://academysoftwarefdn.slack.com/messages/CMQ9J4BQC).
+
+## Select One:
+
+[ ] Build problem
+[ ] Incorrect Functionality or bug
+[ ] New feature or functionality
+
+## Description
+
```

Comment:
```suggestion
<!-- Please try to describe your issue as precisely as possible so we better can understand your suggestion/situation -->


```
Add some air to better highlight the description header. 
Should we also add a comment to encourage a precise description?
Wording of comment needs work..

Reply:
How about

Please make the description as precise as possible so we can best understand your suggestion or issue.

Reply:
I like it!